pypose/lietensor/lietensor.py

```diff
@@ -27,6 +32,17 @@
                      'index_select', 'masked_select', 'index_copy', 'index_copy_',
                      'select', 'select_scatter', 'index_put','index_put_', 'copy_']
 
+_size_any_t = Union[float, Tuple[float, ...], torch.Tensor]
```

Comment:
remove `_size_any_t`, you don't show it in docs, so it is unnecessary.

Reply:
done